os/linux/io_uring.h

```diff
@@ -1,4 +1,4 @@
-/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
+/* SPDX-License-Identifier: (GPL-2.0 WITH Linux-syscall-note) OR MIT */
```

Comment:
This change seems unrelated to zone append and probably should be a different patch with a commit message justifying it.

Reply:
This file will be sent as a separate patch, not part of ZNS.